core/bentley/src/Tracing.ts

```diff
@@ -22,6 +23,8 @@ export enum SpanKind {
   CONSUMER = 4
 }
 
+/* eslint-disable deprecation/deprecation */
```

Comment:
Please use `eslint-disable-line`/`next-line`.

Reply:
the Tracing class has 29 references to itself, can we make an exception for this file? 😅 I moved it closer to the class to reduce collateral damage, let me know if that's ok for you